tmail-backend/jmap/extensions/pom.xml

```diff
@@ -20,10 +20,16 @@
             <artifactId>team-mailboxes</artifactId>
         </dependency>
         <dependency>
-            <groupId>${james.groupId}</groupId>
-            <artifactId>apache-james-backends-es-v7</artifactId>
+            <groupId>org.elasticsearch.client</groupId>
+            <artifactId>elasticsearch-rest-high-level-client</artifactId>
+            <version>7.10.2</version>
```

Comment:
Should we have a {property} or add version in parent pom?

Reply:
For this one, I think it is no. Cause we have another version 6.4.3 for backport.